app/assets/stylesheets/colors.css.scss

```diff
@@ -7,7 +7,7 @@ $background-blue: #E7F2F7;
 $grey: #2B2B2B;
 $light-grey: #DDDDDD;
 
-$border-grey: #DDDDDD;
+$border-grey: #CCCCCC;
```

Comment:
I just did some tries on the profile page and the stream and a little darker feels better to me.
If we think that this change should be a separate PR, I can drop it from this one.